### PR TITLE
FIX: nullable reference

### PR DESCRIPTION
**Pull request by davcrypto**

### CODE REVIEW

File: usecase/parser/msg.go [hunk 1]
```diff
@@ -1235,6 +1241,12 @@ func parseRawMsgGenericGrant(
 		MaybeGenericGrant: &rawMsg,
 	}
 
+	// Getting possible signer address from Msh
```
Comment (calvinaco) [inline usecase/parser/msg.go:1250]:
```suggestion
	// Getting possible signer address from Msg
```

File: usecase/parser/msg.go [hunk 2]
```diff
@@ -152,6 +152,12 @@ func ParseBlockTxsMsgToCommands(
 func ParseMsgSend(
 	parserParams utils.CosmosParserParams,
 ) ([]command.Command, []string) {
+	// Getting possible signer address from Msh
```
Comment (calvinaco) [inline usecase/parser/msg.go:155]:
```suggestion
	// Getting possible signer address from Msg
```

File: usecase/parser/msg.go [hunk 3]
```diff
@@ -152,6 +152,12 @@ func ParseBlockTxsMsgToCommands(
...
+	var possibleSignerAddresses []string
+	if parserParams.Msg["from_address"] != nil {
+		possibleSignerAddresses = append(possibleSignerAddresses, parserParams.Msg["from_address"].(string))
+	}
```
Comment (calvinaco) [inline usecase/parser/msg.go:159]: Do we need to cater invalid MsgSend here?

**Review by calvinaco — CHANGES_REQUESTED** [review]

File: usecase/parser/msg.go (hunk 3 above)
Comment (davcrypto) [reply usecase/parser/msg.go:159]: right, updated

File: usecase/parser/msg.go [hunk 4]
```diff
@@ -152,6 +152,12 @@ func ParseBlockTxsMsgToCommands(
 func ParseMsgSend(
 	parserParams utils.CosmosParserParams,
 ) ([]command.Command, []string) {
+	// Getting possible signer address from Msg
+	var possibleSignerAddresses []string
+	if parserParams.Msg != nil && parserParams.Msg["from_address"] != nil {
```
Comment (calvinaco) [inline usecase/parser/msg.go:157]: I am actually wondering if `parserParams.Msg["from_address"]` is `nil`able

Comment (davcrypto) [reply usecase/parser/msg.go]: Thanks for remind

**Review by calvinaco — APPROVED** [review]

**Pull request merged**